insert.go

```diff
@@ -92,7 +98,11 @@ func (stmt *InsertStmt) OnConflict(clause *ConflictClause) *InsertStmt {
 // bindings. It is used internally by Exec, GetRow and GetAll, but is
 // exported if you wish to use it directly.
 func (stmt *InsertStmt) ToSQL(rebind bool) (asSQL string, bindings []interface{}) {
-	var clauses = []string{"INSERT INTO " + stmt.Table}
+	var clauses = []string{"INSERT", "INTO", stmt.Table}
+
+	if stmt.ignoredIfExists && len(clauses) > 0 {
```

Comment:
No real reason to check if `len(clauses) > 0`, we set it in the previous line so it can't be empty.